assertk-coroutines/build.gradle

```diff
@@ -0,0 +1,336 @@
+                allWarningsAsErrors = true
+                freeCompilerArgs += ["-Xopt-in=kotlin.RequiresOptIn"]
+            }
+        }
+    }
+
+    def kotlin_coroutines_version = '1.3.4'
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation kotlin('stdlib-common')
+                implementation project(':assertk')
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core-common:$kotlin_coroutines_version"
+            }
+        }
+        commonTest {
+            dependencies {
+                implementation kotlin('test-common')
+                implementation kotlin('test-annotations-common')
+            }
+        }
+        jvmMain {
+            dependencies {
+                implementation kotlin('stdlib-jdk8')
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlin_coroutines_version"
+            }
+        }
+        jvmTest {
+            dependencies {
+                implementation project(':java-interop')
+                implementation kotlin('test-junit')
+                implementation kotlin('reflect')
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlin_coroutines_version"
+            }
+        }
+        jsMain {
+            dependencies {
+                implementation kotlin('stdlib-js')
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core-js:$kotlin_coroutines_version"
+            }
+        }
+        jsTest {
+            dependencies {
+                implementation kotlin('test-js')
+            }
+        }
+        nativeMain {
+            dependsOn(commonMain)
+        }
+        nativeTest {
+            dependsOn(commonTest)
+        }
+        [linuxMain, iosArm64Main, iosX64Main, macosMain].each {
+            it.dependsOn(nativeMain)
+        }
+        [linuxTest, iosArm64Test, iosX64Test, macosTest].each {
+            it.dependsOn(nativeTest)
+        }
+        linuxMain {
+            dependencies {
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core-linuxx64:$kotlin_coroutines_version"
+            }
+        }
+        iosArm64Main {
+            dependencies {
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core-iosarm64:$kotlin_coroutines_version"
+            }
+        }
+        iosX64Main {
+            dependencies {
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core-iosx64:$kotlin_coroutines_version"
+            }
+        }
+        macosMain {
+            dependencies {
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core-macosx64:$kotlin_coroutines_version"
+            }
+        }
+    }
+}
+
+task nativeTest {
+    dependsOn(linuxTest, macosTest, iosX64Test)
+}
+
+task test {
+    dependsOn(jvmTest, jsTest, nativeTest)
+}
+
+task detektMain(type: io.gitlab.arturbosch.detekt.Detekt) {
```

Comment:
maybe import `io.gitlab.arturbosch.detekt.Detekt`

Reply:
Most of this was duplicated from the assertk build.gradle, I can do a follow-up pr to clean them up.